src/Websites/Websites/Utilities/KeyVaultClient.cs

```diff
@@ -25,11 +25,21 @@ public KeyVaultManagementClient WrappedKeyVaultClient
             private set;
         }
 
-        public Vault GetKeyVault(string resourceGroupName, string vaultName)
+        public Vault GetKeyVault(string resourceGroupName, string vaultName, string kvSubscriptionId)
```

Comment:
```suggestion
        public Vault GetKeyVault(string resourceGroupName, string vaultName, string kvSubscriptionId = null)
```
To keep compatibility with previous codes.

Reply:
Done!